core/templates/dev/head/pages/exploration_player/TutorCardDirective.js

```diff
@@ -148,6 +161,12 @@ oppia.directive('tutorCard', [
             });
           };
 
+          $scope.isContentAudioTranslationAvailable = function() {
+            return Object.keys(oppiaPlayerService
```

Comment:
No, I meant, this could be: `return oppiaPlayerService.isContentAudioTranslationAvailable(...)`.